checkov/common/output/csv.py

```diff
@@ -51,7 +51,7 @@ def __init__(self) -> None:
     def add_report(self, report: Report, git_org: str, git_repository: str) -> None:
         if report.check_type in (CheckType.SCA_PACKAGE, CheckType.SCA_IMAGE):
             for record in itertools.chain(report.failed_checks, report.passed_checks, report.skipped_checks):
-                if record.check_name == SCA_PACKAGE_SCAN_CHECK_NAME:
+                if record.check_name != SCA_LICENSE_CHECK_NAME:
```

Comment:
@NoaAzoulay In general, allow-listing is much better for consistent behavior than block listing. Can we use `in (...)` instead of this not-equals?